contrib/linux/requirements.txt

```diff
@@ -1,3 +1,3 @@
 # used by file watcher sensor
-pyinotify>=0.9.5,<=0.10
--e git+https://github.com/Kami/logshipper.git@stackstorm_patched#egg=logshipper
+git+https://github.com/blag/watchdog3.git#python2.7-support
+pathtools3>=0.2.0
```

Comment:
It looks like we don't need to specify this here since it's already specified in `watchdog3` setup.py file.